submissions/Valkirin/pop-up/source.css

```diff
@@ -0,0 +1,136 @@
+html { overflow-y: scroll; }
+
+* { margin: 0; padding: 0; }
+
+body { 
+	font: 16px Arial, serif; 
+	color: #fff
+}
+
+nav {display: block;}
+
+label {	position: relative;}
+
+ul {list-style-type: none;}
```

Comment:
Avoid using styling by tags. It's ok for `html` and `body`, but not for other elements. Use classes or id's instead.

Reply:
Thank you. I will try not to do so.

Reply:
I saw your comment about tags too late. Now I will try to fix it.

Reply:
Fixed